### PR TITLE
add additional info to the migration docs

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
Adds additional info to the migration docs

- When to migrate
- shaders
- filters
- Textures

### CODE REVIEW

File: docs/guides/migrations/v8.md [hunk 1]
```diff
@@ -112,6 +140,46 @@ PixiJS will now need to be initialised asynchronously. With the introduction of
...
+
+```
+
+const image = new Image();
+
+image.onload = function(){
+
+  // create a texture source
+  const source = new ImageSource({
+    resource
```
Comment (bigtimebuddy) [inline docs/guides/migrations/v8.md]: Is this what you meant here?

```suggestion
    resource: image,
```

File: docs/guides/migrations/v8.md [hunk 2]
```diff
@@ -251,6 +319,101 @@ const graphics2 = new Graphics()
...
+  uTime:{value:1, type:'f32',
+});
+
+uniformGroup.uniforms.uTime = 100; // still accessed the same!
+```
+
+The best way to play and fully and get to know this new setup please check out the Mesh and Shader examples:
+
+**old**: [v7 example](https://pixijs.com/8.x/examples/mesh-and-shaders/triangle-color)
+**new**: [v8 example](https://pixijs.com/7.x/examples/mesh-and-shaders/triangle-color)
```
Comment (bigtimebuddy) [inline docs/guides/migrations/v8.md]: These are mixed up:

```suggestion
**old**: [v7 example](https://pixijs.com/7.x/examples/mesh-and-shaders/triangle-color)
**new**: [v8 example](https://pixijs.com/8.x/examples/mesh-and-shaders/triangle-color)
```

File: docs/guides/migrations/v8.md [hunk 3]
```diff
@@ -251,6 +319,101 @@ const graphics2 = new Graphics()
...
+        }),
+        resources: {
+            timeUniforms: {
+                uTime: { value: 0.0, type: 'f32' },
+            },
+        },
+    });
+```
+**old**: [v7 example](https://pixijs.com/8.x/examples/filters-advanced/custom)
+**new**: [v8 example](https://pixijs.com/7.x/examples/filters-advanced/custom)
```
Comment (bigtimebuddy) [inline docs/guides/migrations/v8.md]:
```suggestion
**old**: [v7 example](https://pixijs.com/7.x/examples/filters-advanced/custom)
**new**: [v8 example](https://pixijs.com/8.x/examples/filters-advanced/custom)
```

**Review by bigtimebuddy — APPROVED** [review]

File: docs/guides/migrations/v8.md [hunk 4]
```diff
@@ -17,6 +17,34 @@ PixiJS v8 introduces several exciting changes and improvements that dramatically
...
+ - Storybook
+ - UI
+ -  - Open Games
+
+**Migrating Right Now:**
+ - React
+ - Spine (esoteric version)
+
+**To Be Migrated:**
+ - Pixi viewport
```
Comment (Zyie) [inline docs/guides/migrations/v8.md]:
```suggestion
```

**Pull request merged**